pkg/bpf/map_linux.go

```diff
@@ -286,7 +319,7 @@ func (m *Map) updatePressureMetric() {
 		return
 	}
 
-	pvalue := float64(len(m.cache)) / float64(m.MaxEntries)
+	pvalue := float64(len(m.cache)) / float64(m.MaxEntries())
```

Comment:
This could occur before this change as well, I don't think we ever validated this piece of input. Seeing as `updatePressureMetric()` is only ever invoked after calls to `Map.open()`, which is guaranteed to fail with maxentries 0, I don't think this is an issue.